### PR TITLE
Updating the Python SDK docs

### PR DESCRIPTION
**Pull request by andher1802**
Updating the python SDK docs for the getting stated section. 

Items:
* [ x] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ x] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

File: docs/30-second-example.md [hunk 1]
```diff
@@ -7,25 +7,41 @@ The UP42 Python package uses nine classes, representing the **hierarchical struc
...
 workflow = project.create_workflow(name="30-seconds-workflow", use_existing=True)
 #print(up42.get_blocks(basic=True))
 input_tasks = ["Sentinel-2 L2A Visual (GeoTIFF)",
                "Sharpening Filter"]
 workflow.add_workflow_tasks(input_tasks)
+```
+
+### **Defining the input parameters**
+
+Input tasks are not enough to completely define our workflow. Also, input parameters should be defined which include the AOI (area of interest), dates of analysis, among others. The AOI can be added into the workflow by different methods, external geometry files, and operations as well as some example AOI included in our package.
```
Comment (chrieke) [inline docs/30-second-example.md:42]:
```suggestion
To further configure the workflow, we provide workflow input parameters, e.g. the AOI (area of interest), dates of analysis, among others. The AOI geometry can be read via various methods (e.g. `up42.draw_aoi()`, `up42.read_vector_file()`) and provided as multiple formats (e.g. GeoJSON, shapely Polygon, GeoDataFrame).
```

File: docs/30-second-example.md [hunk 2]
```diff
@@ -35,21 +51,42 @@ input_parameters = workflow.construct_parameters(geometry=aoi,
                                                  end_date="2020-12-31",
                                                  limit=1)
 input_parameters["esa-s2-l2a-gtiff-visual:1"].update({"max_cloud_cover":5})
+```
+
```
Comment (chrieke) [inline docs/30-second-example.md:55]:
```suggestion
### **Price Estimation and Pre-testing**
```

File: docs/30-second-example.md [hunk 3]
```diff
@@ -35,21 +51,42 @@ input_parameters = workflow.construct_parameters(geometry=aoi,
...
+Price estimation is an important feature that allows to estimate the actual costs of your workflow before you run your application. Estimating costs are reported in credits units that you can check on your account dashboard.
 
+```python
 # Price estimation
 workflow.estimate_job(input_parameters)
+```
+
+### **Pre-testing**
```
Comment (chrieke) [inline docs/30-second-example.md:63]:
```suggestion
```

File: docs/30-second-example.md [hunk 4]
```diff
@@ -35,21 +51,42 @@ input_parameters = workflow.construct_parameters(geometry=aoi,
...
+Prior a final run of our workflow we can test the configuration and availability of the tasks included in our pipeline. This is done by running test jobs before the final run.
 
+```python
 # Run a test job to query data availability and check the configuration.
 test_job = workflow.test_job(input_parameters, track_status=True)
+```
+
 
+### **Final run**
```
Comment (chrieke) [inline docs/30-second-example.md:72]:
```suggestion
### **Running the Workflow**
```

**Review by chrieke — COMMENTED** [review]
Great work, really like the new structure and the additional explanations & visualizations make it much more accessible! Made some optional suggestions to condense or clarify some of the explanations a bit more.

File: docs/installation.md [hunk 5]
```diff
@@ -11,6 +11,8 @@ Update an existing installation to the **newest version** via:
 ```bash
 pip install up42-py --upgrade
 ```
+!!! Note
+    The latest development version of Up42 is on [GitHub](https://github.com/up42/up42-py).
```
[the remark this answers is not in the document]
Comment (andher1802) [reply docs/installation.md:15]: Sure. I was not aware of the GitHub PyPi difference until you mentioned yesterday. We can keep it this way until we further develop a release strategy including the conda package as well.

File: docs/authentication.md [hunk 6]
```diff
@@ -12,32 +12,36 @@ In the project's **Developers section** you can find the **project_id** and **pr
...
 ## Authenticate  
 
-Authenticate by providing the project credentials **directly in the code**:
+Authenticate by providing the project credentials **directly in the code**, (note that the project ID string and the project-API-key come from the your account credentials in the previous step):
 
-```python
+``` py title="Inline authentication"
 import up42
-up42.authenticate(project_id="123", project_api_key="456")
+up42.authenticate(project_id="project ID string", project_api_key="project-API-key")
```
[the remark this answers is not in the document]
Comment (andher1802) [reply docs/authentication.md:19]: Done!!!!

File: docs/catalog.md [hunk 7]
```diff
@@ -4,15 +4,21 @@
...
 ## Initialize Catalog
 
 ```python
 import up42
 up42.authenticate(project_id="123", project_api_key="456")
-#up42.authenticate(cfg_file="config.json")
+catalog = up42.initialize_catalog()
+```
+
+Alternatively, if you want to authenticate with a config file (make sure config.json file is on the current folder or set the path)
```
[the remark this answers is not in the document]
Comment (andher1802) [reply docs/catalog.md:17]: Sure... I agree...

**Review by chrieke — APPROVED** [review]

**Pull request merged**